Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -73,6 +78,19 @@ private extension LayoutManager {
                 let blockquoteRect = self.blockquoteRect(origin: origin, lineRect: rect, blockquoteIndent: blockquoteIndent, lineEndsParagraph: lineEndsParagraph)
 
                 self.drawBlockquote(in: blockquoteRect.integral, with: context)
+                
+                //nested blockquote
+                let nestDepth = paragraphStyle.blockquoteNestedIndent.depth
+                guard nestDepth >= 1 else {return}
+                for index in 1...nestDepth {
+                    let indent = CGFloat(index) * Metrics.listTextIndentation
+                    
+                    let nestRect = self.blockquoteRect(origin: origin, lineRect: rect, blockquoteIndent: indent, lineEndsParagraph: lineEndsParagraph)
+                    
+                    self.drawNestedBlockquote(in: nestRect.integral, with: context, at: index)
+                }
+                
+            
```

Comment:
Instead of having this two different ways of drawing the borders I suggest we do the following changes:

 - Rename the method `drawBlockquote()` to be `drawBlockquoteBackground()` and change it to only draw the background of the blockquote.
 - Rename the `drawNestedBlockquote()` to be `drawBlockquoteBorder()` and use it to draw **all** the borders of the blockquote.

This will simplify the code and it make more obvious the steps involved.

Reply:
oh good idea